### PR TITLE
Create meta.yaml

### PR DESCRIPTION
**Pull request by agcopenhaver**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0

### CODE REVIEW

**Comment by agcopenhaver** [comment]
not building on windows because no scipy

**Comment by scopatz** [comment]
hmm there is definitely scipy on Windows

**Comment by agcopenhaver** [comment]
@scopatz this recipe requires scipy 1.3 which I’m not seeing in the channel as being available for windows. Am I mistaken?

**Comment by scopatz** [comment]
Woah weird...

**Comment by agcopenhaver** [comment]
@conda-forge-admin, please rerender

**Comment by agcopenhaver** [comment]
@scopatz until scipy 1.3 is available for windows, which seems to be on the way, would it be okay to disable windows and juist build for osx and linux?

File: recipes/creme/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    - creme.preprocessing
+    - creme.proba
+    - creme.reco
+    - creme.stats
+    - creme.tree
+    - creme.utils
+
+about:
+  home: "https://github.com/creme-ml/creme"
+  license: "BSD"
```
Comment (chrisburr) [inline recipes/creme/meta.yaml]: Which BSD licence is it? At at glance it looks like this but please check
```suggestion
  license: BSD-3-Clause
```

File: recipes/creme/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  sha256: "109e308070e023825efe0b3eb6f94898e59a24ce52da259828b2eb473dfed4c6"
+
+build:
+  skip: True  # [win or py<35]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - numpy >=1.16.4
```
Comment (chrisburr) [inline recipes/creme/meta.yaml]: Do you need most of these in `host`? It looks like `python` and `pip` will be enough (these are only required to run the `setup.py` script).

File: recipes/creme/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/creme-ml/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: "109e308070e023825efe0b3eb6f94898e59a24ce52da259828b2eb473dfed4c6"
+
+build:
+  skip: True  # [win or py<35]
```
Comment (chrisburr) [inline recipes/creme/meta.yaml]: Can this be `noarch`? It's fine if Windows then fails due to missing dependencies as the feedstock will only build for Linux.
```suggestion
  noarch: python
```
The python version constraint can be added using `python >=3.6` in both `host` and `run`.

File: recipes/creme/meta.yaml (hunk 1 above)
Comment (agcopenhaver) [reply recipes/creme/meta.yaml]: it's bsd-3, will change

**Comment by agcopenhaver** [comment]
@chrisburr builds fine for linux and osx with your proposed edits

**Pull request merged**